pyproject.toml

```diff
@@ -2,7 +2,7 @@
 name = "chat-history"
 version = "0.1.0"
 description = ""
-authors = ["Dima Kuchin <dima@mirable.io>"]
+authors = ["Abdullah Almahfadi <abalmahfadi@gmail.com>"]
```

Comment:
why replace? you're not the original author

Reply:
Oh sorry I removed it. But If you didn't mind I want to keep it in my own version of project because I forward to improve your project more than this little changes.